_docs/bigquery/enabling-billing.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: page
+section: bigquery
+navigation_title: "Enabling billing"
+title: "Enabling billing"
+order: 3
+date: 2020-04-29
+---
+
+<!---
+In this article explained how to Enable BigQuery API if it is not enabled
+-->
+## Before you begin
+* You should have an existing [Google Cloud Platform account ▸](creating-gcp-account)
+
+To use BigQuery with SegmentStream you need an active billing account so that you'll be able to use Google Cloud Platform resources.
+
+**Note: Free Tier Google Cloud accounts don’t support some BigQuery services which are required for SegmentStream Data Import you need to upgrade your account**
+
+## Check if you have an active Free Tier account, and upgrade it if possible:
+On the [main page of Google Cloud Platform](https://console.cloud.google.com) in the header click “Activate” and choose “Upgrade” in the pop-up window:
+![](/img/bigquery_billing.1.png)
+[More information](https://cloud.google.com/free/docs/gcp-free-tier#how-to-upgrade) on upgrading your account.
```

Comment:
More information on upgrading your account you can find [here](https://cloud.google.com/free/docs/gcp-free-tier#how-to-upgrade). - all the sentences should be completed (noun, verb)